src/Skin.js

```diff
@@ -79,7 +92,11 @@ class Skin extends EventEmitter {
      */
     setRotationCenter (x, y) {
         const emptySkin = this.size[0] === 0 && this.size[1] === 0;
-        const changed = x !== this._rotationCenter[0] || y !== this._rotationCenter[1];
+        // Compare a 32 bit x and y value against the stored 32 bit center
+        // values.
+        const changed = (
+            toFloat32(x) !== this._rotationCenter[0] ||
+            toFloat32(y) !== this._rotationCenter[1]);
```

Comment:
What do you think about putting `x = toFloat32(x);` and `y = toFloat32(y);` near the top instead? I suppose the truncation itself isn't a huge expense, but with this formulation I think we'll usually end up doing more conversions than necessary (counting implicit conversions in the assignments below).

| Conversion count | `toFloat32` in `changed` | `x=toFloat32(x)`  |
-|-|-
| x and y unchanged | 2 | 2 |
| x changed, y same | 3 | 2 |
| x same, y changed | 4 | 2 |
| x and y changed | 3 | 2 |